src/api/entities/Participant.ts

```diff
@@ -98,6 +98,17 @@ export const ParticipantCreationPartial = ParticipantSchema.pick({
   users: z.array(UserCreationPartial).optional(),
 });
 
+export const AdminParticipantCreationPartial = z.object({
```

Comment:
I'm finding the naming in this file a bit hard to understand - e.g. what is the relevance of `Admin` in the name here? From the usage, it seems like this one is just to parse the request, whereas `participantCreationAndApprovalPartial` is used to construct the object that we send off to Objection.

Also, I think it might make more sense to move one or both of these elsewhere - I think having the parser live with the entity is, while a pattern we do have, not actually the desired pattern. Per my other comment, I think the goal would to be for the route here to just point to a method in a different file (e.g. https://github.com/IABTechLab/uid2-self-serve-portal/blob/main/src/api/routers/participants/participantsRouter.ts#L540) then the parser could also live in that separate file. 

Does that make sense?

Reply:
Admin* was just bad naming.  We already had a ParticipantCreationPartial so I needed something new but this ended up evolving into something that wasn't really a partial of the entity anymore.  I renamed it ParticipantCreationRequest.

I moved the parsers into participantClasses.ts.